### PR TITLE
Sandworm-valuation-changes

### PR DESCRIPTION
**Pull request by horrible-little-slime**
This has some weird order-of-operations issues with champagne bottle, and fails to solve the issue of the quad-tom

### CODE REVIEW

**Comment by Rinn** [comment]
We can just put the price check on all of the tasks except for tom, I only put it on the quest so i wouldn't duplicate it but w/e.

File: packages/garbo/src/tasks/freeGiantSandworm.ts [hunk 1]
```diff
@@ -293,12 +294,28 @@ function hasWorms(): boolean {
...
+const BASE_RATE = 1 / 100;
+let _sandwormRate: number;
+function sandwormRate(): number {
+  if (!_sandwormRate) {
+    Outfit.from(
+      sandwormSpec(),
+      new Error("Failed to generate Sandworm outfit"),
+    ).dress();
+    _sandwormRate =
+      REJECTION * BASE_RATE * (1 + getModifier("Item Drop") / 100);
```
Comment (Rinn) [inline packages/garbo/src/tasks/freeGiantSandworm.ts:307]: the issue here is the Item Drop modifier is inaccurate until we maximize and buff. actually are we even buffing with the common item drop effects?

Comment (horrible-little-slime) [reply packages/garbo/src/tasks/freeGiantSandworm.ts:307]: I don't think so but great question

Comment (horrible-little-slime) [reply packages/garbo/src/tasks/freeGiantSandworm.ts:307]: I think this isn't crazy accurate but the question is will this be _more_ accurate than what we're currently getting? it looks like we used to use `0.02 * price of melange` and now use `0.01`, which is causing #2138

Comment (Rinn) [reply packages/garbo/src/tasks/freeGiantSandworm.ts:307]: yeah it's fine we should just go with this for now

File: packages/garbo/src/tasks/freeGiantSandworm.ts [hunk 2]
```diff
@@ -228,7 +238,9 @@ const SandwormTasks: GarboFreeFightTask[] = [
     },
     {
       name: "Yellow Ray",
-      ready: () => have($skill`Fondeluge`) || have($item`Jurassic Parka`),
+      ready: () =>
+        (drumMachineWorthIt() && have($skill`Fondeluge`)) ||
+        have($item`Jurassic Parka`),
```
[the remark this answers is not in the document]
Comment (Rinn) [reply packages/garbo/src/tasks/freeGiantSandworm.ts:243]: minor logic error here right?

**Review by Rinn — APPROVED** [review]

**Pull request merged**